src/bindings/c/src/ov_core.cpp

```diff
@@ -324,18 +324,16 @@ void ov_core_versions_free(ov_core_version_list_t* versions) {
 
 #ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
 ov_status_e ov_core_create_with_config_unicode(const wchar_t* xml_config_file_ws, ov_core_t** core) {
-    if (!core || !xml_config_file_ws) {
+    if (!xml_config_file_ws) {
         return ov_status_e::INVALID_C_PARAM;
     }
 
+    std::string xml_config_file;
```

Comment:
First of all I would like to avoid code duplication in method: ov_core_compile_model_from_file_unicode
in order to have logic with `va_args` in one place. Is it possible?

Reply:
It seems it is hard, because va_args mechanism doesn't support nested calling. It means va_args parameter into one function cannot be passed to its inner called function.